pkg/network/openshift_sdn.go

```diff
@@ -9,11 +11,17 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/bootstrap"
+	types "k8s.io/apimachinery/pkg/types"
```

Comment:
This grouping is weird.  `operv1 "github.com/openshift/api/operator/v1"` should be grouped with the other openshift/api imports as it was before, `"github.com/openshift/cluster-network-operator/pkg/bootstrap"` and `"k8s.io/klog/v2"` probably should be in separate groups, and `types "k8s.io/apimachinery/pkg/types"` can be grouped with the other apimachinery imports.